poglink/__init__.py

```diff
@@ -1 +1,7 @@
+from importlib.metadata import version, PackageNotFoundError
 from . import cogs
+
+try:
+    __version__ = version("poglink")
+except PackageNotFoundError:
+    pass
```

Comment:
Allows you to do:
```python
import poglink 

print(poglink.__version__)
```

See [here](https://github.com/pypa/setuptools_scm#retrieving-package-version-at-runtime) for reference